App/App.xaml.cs

```diff
@@ -281,6 +279,7 @@ public bool RecoverUserInfo()
         // Set Userdata object
         return (SaveInfo = DataFetcher.UserData = JsonConvert.DeserializeObject<User>(userDataStr)) != null;
     }
+
```

Comment:
I'm leaving the clean-up-related items here. Since the PR aims at improving maintainability, it should include code cleaning